.nvmrc

```diff
@@ -1 +1 @@
-12.16.1
+16.17.0
```

Comment:
this one is rather huge. Not sure why prebidjs is built against an (for frontend terms) ancient version of node.

I could also upgrade to 20.x , which is the current LTS version, but went for the minimal jump to make the typescript compiler happy. The issue was that some syntax was not supported by default in node 12.x ( `??` operator to be precise )

Reply:
Please see https://github.com/prebid/Prebid.js/pull/10990

Reply:
Ah, related to #10999

Reply:
can we jettison this without breaking things?

Reply:
TIL: jettison :smile: 

Falling back to 12.16.1 , typescript won't transpile files that use newer syntax. There are babel-typescript integrations, but they don't typecheck, which defeats a big purpose and the gulp integration is actually trivial.

I guess minimal typescript support is fine to wait for the prebid 9 release, though.

Or maybe I got the meaning of jettison wrong :sweat: